CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.2 (2020-02-11)
+
+- Fixed the message displayed for upgrading to the latest version. [#61](https://github.com/blackbaud/skyux-sdk-cli/pull/61)
```

Comment:
What do we mean by `displayed for upgrading`? Do we mean that "that is displayed when users upgrade"? Or do we mean "that prompts users to upgrade"? Or something different?

Reply:
The latter.  Original issue: https://github.com/blackbaud/skyux-sdk-cli/issues/60